apm-agent-plugins/apm-api-plugin/pom.xml

```diff
@@ -20,7 +20,7 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>apm-agent-api</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
Let's add the new functionality without adding this dependency

Reply:
Done